charts/dnsmadeeasy-webhook/README.md

```diff
@@ -0,0 +1,135 @@
+```
+
+## Custom configuration
+
+N/A
+
+## Values
+
+**Important**: When deploying an application Helm chart you can add more values from our common library chart [here](https://github.com/k8s-at-home/charts/tree/master/charts/common/)
+
+| Key | Type | Default | Description |
+|-----|------|---------|-------------|
+| additionalVolumeMounts[0].mountPath | string | `"/tls"` |  |
+| additionalVolumeMounts[0].name | string | `"certs"` |  |
+| additionalVolumeMounts[0].readOnly | bool | `true` |  |
+| args | string | `"[\"--tls-cert-file=/tls/tls.crt\",\"--tls-private-key-file=/tls/tls.key\"]"` |  |
+| certManager.namespace | string | `"cert-manager"` | Namespace where the cert-manager operator was installed to |
+| certManager.serviceAccountName | string | `"cert-manager"` | Service account used by the cert-manager |
+| groupName | string | `"acme.mycompany.com"` | The GroupName here is used to identify your company or business unit that created this webhook. This name will need to be referenced in each Issuer's `webhook` stanza to inform cert-manager of where to send ChallengePayload resources in order to solve the DNS01 challenge. This group name should be **unique**, hence using your own company's domain here is recommended. |
+| image.pullPolicy | string | `"IfNotPresent"` | Image pull policy |
+| image.repository | string | `"ghcr.io/k8s-at-home/dnsmadeeasy-webhook"` | Image repository |
+| image.tag | string | `"v1.0.0"` | Image tag |
+| probes.liveness.custom | bool | `true` |  |
+| probes.liveness.enabled | bool | `true` |  |
+| probes.liveness.spec.httpGet.path | string | `"/healthz"` |  |
+| probes.liveness.spec.httpGet.port | string | `"https"` |  |
+| probes.liveness.spec.httpGet.scheme | string | `"HTTPS"` |  |
+| probes.readiness.custom | bool | `true` |  |
+| probes.readiness.enabled | bool | `true` |  |
+| probes.readiness.spec.httpGet.path | string | `"/healthz"` |  |
+| probes.readiness.spec.httpGet.port | string | `"https"` |  |
+| probes.readiness.spec.httpGet.scheme | string | `"HTTPS"` |  |
+| probes.startup.custom | bool | `true` |  |
+| probes.startup.enabled | bool | `true` |  |
+| probes.startup.spec.httpGet.path | string | `"/healthz"` |  |
+| probes.startup.spec.httpGet.port | string | `"https"` |  |
+| probes.startup.spec.httpGet.scheme | string | `"HTTPS"` |  |
+| service.port.name | string | `"https"` |  |
+| service.port.port | int | `443` |  |
+
+## Changelog
+
+All notable changes to this application Helm chart will be documented in this file but does not include changes from our common library. To read those click [here](https://github.com/k8s-at-home/charts/tree/master/charts/common/README.md#Changelog).
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+### [2.0.0]
+
+#### Added
+
+- N/A
+
+#### Changed
+
+- Moved chart to k8s-at-home
+
+#### Removed
+
+- N/A
+
+[2.0.0]: #2.0.0
```

Comment:
This should probably be in sync with the CHANGELOG.md file?

Reply:
README.md is generated